modules/bibdk_reviews/bibdk_reviews.article_review.inc

```diff
@@ -96,7 +96,7 @@ function bibdk_reviews_article_view($manifestation) {
   }
 
   $rating = $manifestation->getReviewRatings();
-  $title_array = _ting_openformat_parse_element($manifestation->getTitle());
+  $title_array = _ting_openformat_parse_element($manifestation->getTitleSpecific());
```

Comment:
Levere getTitleSpecific() altid noget (f.eks. streng) og hvad forventer _ting_openformat_parse_element af input?
Stemmer de to overens?

Reply:
godt spørgsmål... det er jo noget der har været der hele tiden

Reply:
Så må der jo være tænkt over det :)